rest_framework_simplejwt/serializers.py

```diff
@@ -73,6 +74,9 @@ def validate(self, attrs):
         data['refresh'] = str(refresh)
         data['access'] = str(refresh.access_token)
 
+        if api_settings.UPDATE_LAST_LOGIN:
+            update_last_login(None, self.user)
+
```

Comment:
I am thinking people might expect it to work with the `TokenObtainSlidingSerializer` as well.

Reply:
+1 Adding it to sliding serializer would be helpful.

Reply:
Added to `TokenObtainSlidingSerializer`. Please review and verify as I have not used `TokenObtainSlidingSerializer` before.